src/lib/canvas/groupExport.ts

```diff
@@ -0,0 +1,17 @@
+import {TeamSet} from "@/types/TeamSet"
+
+/**
+ * Export a TeamSet to a CSV format compatible with Canvas LMS.
+ * The generated CSV includes columns for canvas_user_id and group_name.
+ *
+ * @param {TeamSet} teamSet - The TeamSet to be exported.
+ * @returns {string} The CSV representation of the TeamSet for Canvas.
+ */
+export function exportTeamSetToCanvas(teamSet: TeamSet): string {
+    let exportedCSV = teamSet.teams.map((team, index) => {
+        const teamName = team.name || `Group ${index + 1}`
+        return team.students.map(student => `${student.id},${teamName}`).join("\n")
```

Comment:
As something to ponder, this does imply that student ids need to be the canvas id. This could be a separate thing in the object or student id might need to be a string?

Throwing this out because idk

Reply:
Hmm, yeah I think it should be that the canvas id should maybe be separate from the actual id if we are trying to decouple from canvas. What this means for the database schema though, I'm not sure. Would we have an `id` that we generate, but then also an `imported_id` that is from the csv import process in #8? That way it's not specific to Canvas, but could be adapted to other systems such as Moodle

Reply:
Yeah actually let's bring this up in chat/meeting - I'm on board with this, some kind of `lms_id` in the schema